x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules/persistence_google_workspace_role_modified.json

```diff
@@ -50,5 +50,5 @@
   ],
   "timestamp_override": "event.ingested",
   "type": "query",
-  "version": 10
```

Comment:
This is ok - from the manually reconciled version bumps elastic/detection-rules/pull/1950